ui/package.json

```diff
@@ -43,11 +43,15 @@
     "web-vitals": "^1.1.1",
     "xml-formatter": "^2.6.0"
   },
+  "devDependencies": {
+    "env-cmd": "^10.0.1"
+  },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "craco start",
```

Comment:
isnt it supposed to be `./node_modules/.bin/env-cmd -f .env.basic craco start`?

Reply:
so I'm not sure actually because we have the thing with Mert script which copy env.example to .env
I think we can leave it like that for now and maybe change it after if we will be needed